spec.bs

```diff
@@ -1505,6 +1513,46 @@ partial interface Navigator {
   </wpt>
 </div>
 
+<div algorithm>
+  The <dfn method for=Navigator>deprecatedURNtoURL(|urnOrConfig|, |send_reports|)</dfn>
+  method steps are:
```

Comment:
All of this can be wrapped like so:

```
  The <dfn method for=Navigator>deprecatedURNtoURL(|urnOrConfig|, |send_reports|)</dfn> method
  steps are:
```

and be compliant with 100 lines.